cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -721,7 +720,7 @@ var CMS = window.CMS || {};
                 debug.on(this.mouseEnter + ' ' + this.mouseLeave, function (e) {
                     clearTimeout(timer);
 
-                    if (e.type === that.mouseEnter) {
+                    if (e.type === 'mouseenter') {
```

Comment:
yes, it works
it's not weird :)
when event happens it has the type, and the type is always the _original_ event name, which in this case is `mouseenter`. otherwise we were always comparing `'mouseenter' === 'mouseenter.cms.toolbar'` which in turn never showed the message